src/ensembl/src/shared/components/layout/StandardAppLayout.tsx

```diff
@@ -26,6 +26,7 @@ type StandardAppLayoutProps = {
   sidebarToolstripContent?: ReactNode;
   sidebarNavigation: ReactNode;
   topbarContent: ReactNode;
+  isSidebarFloating?: boolean;
```

Comment:
Since this prop is added to defaultProps below, it will always present on the Props type; which means that it's a non-optional field.

Reply:
✔️